AdaptiveCards/rendering-cards/renderer-status.md

```diff
@@ -33,8 +33,8 @@ The tables below show the current status of each renderer, based on their public
 |--- | --- | --- | --- | --- | --- | --- |
 |Spacing and Separator | ✅ | ✅ | ✅ | ✅ | ✅ |
 |[TextBlock DATE/TIME formatting](../authoring-cards/text-features.md#datetime-formatting-and-localization) | ✅ | ✅ | ✅ | ✅ | ✅ |
-|[TextBlock Markdown support](../authoring-cards/text-features.md#markdown) | ✅* | ✅ | ✅ | ✅ | ✅ |
-|Full Input support | ✅ | ✅ | ✅ | ✅ | ✅ |
+|[TextBlock Markdown support](../authoring-cards/text-features.md#Markdown-Commonmark-subset) | ✅* | ✅ | ✅ | ✅ | ✅ |
```

Comment:
>* [](start = 97, length = 1)

This might have been recently updated with Matt's changes perhaps - Pls update link to "#markdown-commonmark-subset"